projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts

```diff
@@ -108,6 +114,34 @@ export class SequenceBarRendererService {
       .classed('bar-value', true);
   }
 
+  private drawBarMarkers(
+    plotSelection: SequenceSVGSelection,
+    xScale: ScaleLinear<number, number>,
+    options: SequenceOptions
+  ): void {
+    let index = 0;
```

Comment:
why not good old for loop with an index instead of managing index separately?

Reply:
donw